scipy/special/_faddeeva.h

```diff
@@ -9,6 +9,7 @@
 #define EXTERN_C_END
 #endif
 
+#include <pyconfig.h>
```

Comment:
Rather than adding this, can you switch the two includes below so `Python.h` comes first?